backend/pkg/helm/release.go

```diff
@@ -373,8 +385,13 @@ func (h *Handler) RollbackRelease(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	err = h.setStatus("rollback", req.Name, processing, nil)
+	if err != nil {
+		zlog.Error().Err(err).Str("action", "rollback").Msg("failed to set status")
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+	}
+
```

Comment:
```suggestion
if err = h.setStatus("rollback", req.Name, processing, nil); err != nil {
		zlog.Error().Err(err).Str("action", "rollback").Msg("setting status")
		http.Error(w, err.Error(), http.StatusInternalServerError)
	}
```

Reply:
updated the error message but didn't change the if block style, as this style is used across.